package.json

```diff
@@ -30,7 +30,7 @@
     "test-ci": "NODE_ENV=test npm run lint && npm run flow && npm run test"
   },
   "dependencies": {
-    "fbjs": "^0.8.7",
+    "fbjs": "^0.8.15",
```

Comment:
LGTM! Maybe worth adding a small polish for line 4: `"version": "0.10.3",`

Reply:
Weird that 0.10.2 is published and this has 0.10.1. I'll double-check nothing slipped through the cracks. Usually the version is bumped automatically when I publish.